### PR TITLE
upgrading radix libraries

### PR DESCRIPTION
**Pull request by brossetti1**
## GitHub Issue

N/A

## Changes

we should probably do some quick testing on this -- the updates are listed here - https://www.radix-ui.com/docs/primitives/overview/releases - some breaking changes we should reveiw

updating radix libraries - they are now on version 1.0.0

discovered this when our tooltip did not match the docs and the interfaces i was trying to access

## Packages Added

Brief list of any packages added, and if folks need to rerun `yarn install` to run locally

## Checks

Before making your PR, please check the following:

- [ ] Critical lint errors are resolved
- [ ] App runs locally
- [ ] App builds locally (run the build command for *any impacted package* and check for any errors before the PR)

### CODE REVIEW

**Comment by 0xRowdy** [comment]
Have you had a chance to test everything out after upgrading?

We can always put a pin in this until after MCon if it has the potential to cause breaking changes.

**Comment by brossetti1** [comment]
@0xRowdy not yet -- yeah i figured we can do it after mcon -- ill add dont merge

**Review by skuhlmann — APPROVED** [review]

**Pull request merged**
